googletest/src/gtest.cc

```diff
@@ -4630,7 +4634,10 @@ void UnitTest::AddTestPartResult(
       // from removing. We use this rather than abort() or __builtin_trap() for
       // portability: Symbian doesn't implement abort() well, and some debuggers
       // don't correctly trap abort().
+
+#ifndef __clang_analyzer__
```

Comment:
... and if __clang_analyzer__ is defined ?
In general this would be the first time we would try to placate a static analysis tool. Not enough reason to make the change here

Reply:
@gennadiycivil If __clang_analyzer__ is defined, you will avoid seeing a warning about undefined behavior that is being intentionally invoked as a workaround for deficiencies of the symbian platform. A better thing to do would be to invoke abort() rather than invoke undefined behavior. I'm happy to do that, but I suspect this will break things downstream eventually (perhaps they should fix abort())?